magma/tuple.py

```diff
@@ -330,6 +332,18 @@ def wire(self, o, debug_info):
                 debug_info=debug_info
             )
             return
+
+        for i, k in enumerate(self.keys()):
+            if not type(self).fields[i].is_wireable(type(o).fields[i]):
+                _logger.error(
+                    WiringLog(f"Cannot wire {{}} (type={type(o)}, to "
+                              f" {{}} (type={type(self)})"
+                              f"because the key {k} is not wireable",
+                              o, self),
+                    debug_info=debug_info
+                )
```

Comment:
```suggestion
                    WiringLog(
                        f"Cannot wire {{}} (type={type(o)}, to "
                        f" {{}} (type={type(self)})"
                        f"because the key {k} is not wireable",
                        o,
                        self
                    ),
                    debug_info=debug_info
                )
```